docs/tutorials/acme/auto-le-cert.rst

```diff
@@ -0,0 +1,149 @@
+========================================================
+Automatically obtain TLS certificates using ingress-shim
+========================================================
+
+The ingress-shim subcomponent of cert-manager can automatically obtain TLS 
+certificates from Let's Encrypt (or another ACME server) through a
+ClusterIssuer resource.
+
+1. Launch cert-manager with defaultIssuer
+
+2. Created an ACME ClusterIssuer
+
+5. Configured cert-manager's :doc:`ingress-shim </reference/ingress-shim>` to
+   automatically provision Certificate resources for all Ingress resources with
+   the ``kubernetes.io/tls-acme: "true"`` annotation, using the ClusterIssuer
+   we have created
+
+6. Verified that the cert-manager installation is working
+
+1. Launch cert-manager
+======================
+
+cert-manager should be deployed using Helm, according to our official
+:doc:`/getting-started/index` guide.
+
+Pick a name for your ClusterIssuer and set it at launch.
+
+.. code-block:: shell
```

Comment:
You need an empty line after the `.. code-block:: shell` lines in order for these to render properly 😬